restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/Call.java

```diff
@@ -2185,6 +2196,8 @@ private void onSipServletResponse(SipServletResponse message, ActorRef self, Act
                         uri.setLrParam(true);
                         challengeRequest.pushRoute(uri);
                     }
+                    //FIXME:should check for the Ims pushed Route?
```

Comment:
is this FIXME applicable anymore?

Reply:
it was present in the old code. didn't check that as it was out of scope